app/forms/hyrax/forms/workflow_action_form.rb

```diff
@@ -34,7 +34,7 @@ def save
         true
       end
 
-      validates :name, presence: true
+      validates :name, :comment, presence: true
```

Comment:
Is a comment always required? Even when it's approved?

Reply:
Perhaps my misunderstanding of the mockup, but I assumed so. Will work thought this.